examples/basic-globalize-compiler/README.md

```diff
@@ -0,0 +1,26 @@
+# Basic Globalize Compiler example
+
+On this example, we assume you know the Globalize usage basics. So, it focuses on
+the Globalize Compiler and on the Globalize runtime modules.
```

Comment:
> Do we have a rule for that somewhere? I'm just curious.

Hard wrapping in markdown is sortof annoying as you can't change a sentence without having to change a bunch of unrelated lines. Though, we don't have a style guide about this, so it boils down to a preference.

Once we had a tiny [discussion](https://github.com/jquery/learn.jquery.com/issues/285#issue-11579167) about it but that's specific for the learn site.